docs/github/getting-started.md

```diff
@@ -84,26 +82,26 @@ jobs:
           lfs: true
 
       # Cache
-      - uses: actions/cache@v1.1.0
+      - uses: actions/cache@v2.1.3
         with:
           path: Library
           key: Library
 
       # Test
       - name: Run tests
-        uses: webbertakken/unity-test-runner@v1.3
+        uses: webbertakken/unity-test-runner@v2
         with:
-          unityVersion: 2019.2.11f1
+          unityVersion: 2019.4.17f1
```

Comment:
With v2, we can leave out unityVersion, as it now defaults to whatever is in their ProjectSettings/ProjectVersiontxt. I think we should encourage using this default.

Reply:
Removed it and mentioned it defaults to `ProjectSettings/ProjectVersion.txt` 👍